packages/components/document-card/src/document-card.js

```diff
@@ -37,14 +38,18 @@ customElementDefineHelper(
 			`;
 		}
 
+		get boldClass() {
+			return this.bold ? 'bold' : '';
```

Comment:
`bold` classname is not a good name for it, the name should be related to a state instead of a style